src/server/metrics.rs

```diff
@@ -2,9 +2,8 @@ use std::net::UdpSocket;
 use std::time::Instant;
 
 use actix_web::{error::ErrorInternalServerError, web::Data, Error, HttpRequest};
-use cadence::{
-    BufferedUdpMetricSink, Counted, Metric, NopMetricSink, QueuingMetricSink, StatsdClient, Timed,
-};
+use cadence::prelude::*;
```

Comment:
We prefer not to do global includes like this for a number of reasons (mostly, it's easier for folk to find the source of a given method or structure if we note where it comes from, avoids some accidental collisions or usages, etc.).
Could you list the structs and methods you're actually using from inside of `cadence::prelude`?

Reply:
Sure, will do.